spec/lib/openid_token_proxy/token/refresh_spec.rb

```diff
@@ -57,12 +57,18 @@ def index
 
     context 'when token was refreshed successfully' do
       it 'executes actions normally returning new tokens as headers' do
-        get :index, refresh_token: refresh_token
-        expect(response).to have_http_status :ok
-        expect(response.body).to eq 'Refresh successful'
-        expect(response.headers['X-Token']).to eq 'new access token'
-        expect(response.headers['X-Refresh-Token']).to eq 'new refresh token'
-        expect(response.headers['X-Token-Expiry-Time']).to eq refreshed_expiry_time.iso8601
+        OpenIDTokenProxy.configure_temporarily do |config|
+          uri = '/#token'
+          block = double('block')
+          config.token_refreshment_hook = proc { |token, error| block.run }
+          expect(block).to receive(:run)
```

Comment:
I realize I didn't [properly spec](https://github.com/hyperoslo/openid-token-proxy/blob/master/spec/controllers/openid_token_proxy/callback_controller_spec.rb#L36-L44) the actual block arguments, but that might be a good idea.

Would something like this work?

```ruby
OpenIDTokenProxy.configure_temporarily do |config|
  expect do |probe|
    config.token_refreshment_hook = probe
    get :index, refresh_token: refresh_token
  end.to yield_with_args(token, null)

  # rest of the expectations
end
```